### PR TITLE
AF-1749: C3 Bubble Chart does not filter data correctly and display an error

### PR DESCRIPTION
**Pull request by jesuino**
Fixing noData state. Now when there's no data the chart won't even attempt to render, instead it will show a no data info. This fixes the bubble chart issue.

### CODE REVIEW

**Comment by jesuino** [comment]
@ederign @jhrcek Could you please review this? The steps to reproduce the issue are in the JIRA, it can be reproduced also on dashbuilder and I think this may be related to what is impeding the integration from work correct (AF-1655) or at least without this fix we won't be able to get AF-1655 merged.

**Comment by jhrcek** [comment]
Just a note: during testing of this I found an old issue (unrelated to this PR - so it doesn't block merging this PR). Just putting it here cause it's related to bubble charts: https://issues.jboss.org/browse/RHPAM-1810

**Comment by jesuino** [comment]
@jhrcek thank you again, Jan!

File: dashbuilder/dashbuilder-client/dashbuilder-renderers/dashbuilder-renderer-c3/src/main/java/org/dashbuilder/renderer/c3/client/C3DisplayerView.java [hunk 1]
```diff
@@ -74,4 +76,22 @@ public void setBackgroundColor(String color) {
                           .setBackgroundColor(color);
     }
     
+    @Override
+    public void noData() {
+        FlowPanel noDataPanel = new FlowPanel();
```
Comment (ederign) [inline dashbuilder/dashbuilder-client/dashbuilder-renderers/dashbuilder-renderer-c3/src/main/java/org/dashbuilder/renderer/c3/client/C3DisplayerView.java]: Hi @jesuino could you please use GWT.create here?

Comment (ederign) [reply dashbuilder/dashbuilder-client/dashbuilder-renderers/dashbuilder-renderer-c3/src/main/java/org/dashbuilder/renderer/c3/client/C3DisplayerView.java]: https://stackoverflow.com/questions/2242399/why-use-gwt-create-instead-of-new

File: dashbuilder/dashbuilder-client/dashbuilder-renderers/dashbuilder-renderer-c3/src/main/java/org/dashbuilder/renderer/c3/client/C3DisplayerView.java [hunk 2]
```diff
@@ -74,4 +76,22 @@ public void setBackgroundColor(String color) {
...
+        noDataPanel.setWidth(width + "px");
+        noDataPanel.setHeight(height + "px");
+        Label lblNoData = new Label(C3DisplayerConstants.INSTANCE.common_noData());
```
Comment (ederign) [inline dashbuilder/dashbuilder-client/dashbuilder-renderers/dashbuilder-renderer-c3/src/main/java/org/dashbuilder/renderer/c3/client/C3DisplayerView.java]: same here.

**Comment by jesuino** [comment]
@ederign resolved conflicts and used GWT.create instead new, thanks!

**Pull request merged**